applications/RANSApplication/custom_python/add_custom_strategies_to_python.cpp

```diff
@@ -16,17 +16,28 @@
 #include <pybind11/pybind11.h>
 
```

Comment:
I'd avoid the `residual_based_whatever` naming as it yields extremely large file names (everything is residual based in Kratos).

Reply:
done :)